Tests/src/PublicAPITests/UserSystemTests.cpp

```diff
@@ -249,6 +263,45 @@ CSP_PUBLIC_TEST(CSPEngine, UserSystemTests, LogInTest)
 	LogOut(UserSystem);
 }
 #endif
+
+#if RUN_ALL_UNIT_TESTS || RUN_USERSYSTEM_TESTS || RUN_USERSYSTEM_LOGIN_AS_GUEST_TEST
+CSP_PUBLIC_TEST(CSPEngine, UserSystemTests, LogInAsGuestTest)
+{
+	auto& SystemsManager = csp::systems::SystemsManager::Get();
+	auto* UserSystem	 = SystemsManager.GetUserSystem();
+
+	csp::common::String UserId;
+
+	// Log in
+	LogInAsGuest(UserSystem, UserId);
```

Comment:
If I have read things correctly LogInAsGuest takes an expected result but none has been provided here. How do we know login was successful?